test/AddressSpaceCast/issue-1341.ll

```diff
@@ -0,0 +1,22 @@
+; RUN: clspv-opt %s -o %t.ll --passes=lower-addrspacecast
+; RUN: FileCheck %s < %t.ll
+
+; CHECK: %0 = phi ptr [ %arrayinit.begin1, %entry ], [ %arrayinit.next4, %arrayinit.body ]
```

Comment:
`%arrayinit.begin1` and `%arrayinit.next4` can easily change, so we should not rely on those hardcoded strings.

Reply:
Fixed. Also added check to ensure the old phi is removed.